template/default.tmpl

```diff
@@ -1,5 +1,5 @@
 {{ define "__alertmanager" }}AlertManager{{ end }}
-{{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts{{ end }}
+{{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts?receiver={{ .Receiver }}{{ end }}
```

Comment:
Wait a sec, that's not generated in the template, isn't it? No magic can know that the receiver param is meant to be regexp escaped.
You mean, that's just in the UI. But the template is escaped for the notification, as said above.
I think we just need `regexp.QuoteMeta` before creating the URL in the template.